src/components/CountriesList.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react'
+import { Link } from 'react-router-dom'
+import axios from 'axios'
+
+export class CountriesList extends Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            countries: []
+        }
+    }
+    async componentDidMount() {
+        const {data: countries} = await axios.get('https://restcountries.eu/rest/v2')
+        this.setState(state => ({
```

Comment:
`this.setState(state => ({countries}))`

Si el value y key tienen mismo nombre no hace falta repetir